bay/plugins/tail.py

```diff
@@ -22,17 +22,18 @@ def load(self):
 @click.option("--host", "-h", type=HostType(), default="default")
 @click.option('--follow/--no-follow', '-f', default=False, help="Follow log output")
 @click.option("--all/--no-all", default=False, help="Show the whole log")
+@click.option("-n", type=int, help="Show specified number of lines")
 @click.argument("container", type=ContainerType())
 @click.argument("lines", default="10")
 @click.pass_obj
-def tail(app, host, container, lines, all=False, follow=False):
+def tail(app, host, container, lines, all=False, follow=False, n=None):
```

Comment:
can you name it `lines_count` or `lines_number` or similar?

one char lines are not a good idea most of the time